.github/workflows/cicd.yml

```diff
@@ -30,10 +30,10 @@ jobs:
           flutter-version: ${{env.FLUTTER_MIN_VERSION}}
       - uses: bluefireteam/melos-action@v3
       - run: melos exec --ignore=flame_3d dart analyze .
```

Comment:
Does flame_3d have to be ignored here now?

Reply:
I don't think we need this line at all. let's see